src/tox_uv/_venv.py

```diff
@@ -30,6 +33,17 @@
     from tox.tox_env.api import ToxEnvCreateArgs
     from tox.tox_env.installer import Installer
 
+INFO_FROM_INTERPRETER_SCRIPT = """
```

Comment:
Do not roll your own, just reuse https://github.com/pypa/virtualenv/blob/main/src/virtualenv/discovery/cached_py_info.py#L28